src/api/app/views/webui/obs_factory/staging_projects/_problems.html.erb

```diff
@@ -41,22 +41,18 @@ end -%>
 <% else %>
   <ul class='problem_list'>
     <%- problems.sort! -%>
-    <%- max = 5 -%>
+    <%- num_elements = 6 -%>
     <%- count = problems.size -%>
-    <% if count <= max+1 # Prevent the "1 more" link %>
-      <% problems.each do |p| %>
-        <%= p %>
-      <% end %>
-    <% else %>
-      <% problems[0,max].each do |p| %>
-        <%= p %>
-      <% end %>
+    <% problems[0, num_elements - 1].each do |p| %>
```

Comment:
`p` doesn't tell much. `problem` would be better. Same for the other each loop below.

Reply:
If problem is an Array you could also write [problems.first(num_elements - 1)](https://ruby-doc.org/core-2.5.0/Array.html#method-i-first)

Reply:
@bgeuken 

> If problem is an Array you could also write problems.first(num_elements - 1) :bowtie:

Yes, it is an array. Good idea! :nerd_face:

Reply:
I think not:
```bash
irb(main):001:0> [1,2,3][0,2]
=> [1, 2]
irb(main):002:0> [1,2,3].first 2
=> [1, 2]
```